apps/docusaurus/docs/create-aptos-dapp/templates/digital-asset.md

```diff
@@ -0,0 +1,190 @@
+
+When you are ready to use a collection on the Public Mint Page, you would want to copy the collection address and assign it to the `collection_id` on the `frontend/config.ts` file.
+
+Some stats are available on this page, like the amount of minted NFTs and Max Supply of the NFTs.
+
+## How can I take it from here?
+
+Remember, one of the goals of this template is to educate and provide a real life example on how a NFT minting dapp can be on Aptos. We provide some basic concepts and features but there is much more you can do for your dapp.
+
+Some ideas you can look at and support are
+
+1. Allowlist mint stage
+2. A custom flow after someone mints a NFT
+3. Check out our [TS SDK](https://github.com/aptos-labs/aptos-ts-sdk) to see what other [API queries](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/digitalAsset.ts) you can use to support more features or fetch more datas
+
+## Ready for MAINNET
+
+If you started your dapp on tesnet and you had all the tests and happy with your NFT collection you probably want to get the collection on mainnet.
+
+Creating a collection on mainnet is the same flow as creating it on tesnet but we need to first configure some things.
+
+1. Change the `VITE_APP_NETWORK` value to `mainnet` on the `.env` file
+2. Run `npm run move:init` to initialize an account to work against the Mainnet
+   1. If you already have an account you would like to use to publish the contract under, you can pass its private key when the prompt asks for that.
+   2. If you are generating a new account, you would want to transfer this account some APT on Aptos Mainnet since the tool can’t fund the account when it is against Mainnet.
+3. Check: open `.aptos/config.yaml` file and see that you have a profile under the `mainnet` name. In addition, open the `.env` file and check the `VITE_MODULE_ADDRESS` value is the same as the mainnet profile account account address.
+4. Head over to `scripts/move/publish.js` file, and change the `minter` address to [`0x3c41ff6b5845e0094e19888cba63773591be9de59cafa9e582386f6af15dd490`](https://explorer.aptoslabs.com/object/0x3c41ff6b5845e0094e19888cba63773591be9de59cafa9e582386f6af15dd490/modules/code/mint_stage?network=mainnet) so it would use the Mainnet token-minter address.
+5. Create or get the account you want to create a collection with, open the `.env` file and assign the account address as the `VITE_CREATOR_ADDRESS` value.
+6. Finally, run `npm run move:publish` to publish your move module on Aptos mainnet.
+7. The next step would be to create a collection using this account. Simply follow [the instructions here](#create-a-nft-collection-page)
+
+## Deploy to a live server
+
+`create-aptos-dapp` utilizes Vite as the development tool. To deploy a Vite static site to a live server, you can simply follow [Vite deployment guide](https://vitejs.dev/guide/static-deploy). In a nutshell, you would need to:
```

Comment:
```suggestion
## Ready for Mainnet

If you started your dapp on testnet, and you are happy with your NFT collection testing, you will want to get the collection on mainnet.

Creating a collection on mainnet is the same flow as creating on testnet, but we need to change some configuration.

1. Change the `VITE_APP_NETWORK` value to `mainnet` in the `.env` file
2. Run `npm run move:init` to initialize an account to work against Mainnet
   1. If you already have an account you would like to use to publish the contract under, you can pass its private key when the prompt asks for that.
   2. If you are generating a new account, you need to transfer this account some APT on Aptos Mainnet since the tool can’t fund the account when it is against Mainnet.
3. Check: open `.aptos/config.yaml` file and see that you have a profile under the `mainnet` name. In addition, open the `.env` file and check the `VITE_MODULE_ADDRESS` value is the same as the mainnet profile account account address.
4. Edit the `scripts/move/publish.js` file, and change the `minter` address to [`0x3c41ff6b5845e0094e19888cba63773591be9de59cafa9e582386f6af15dd490`](https://explorer.aptoslabs.com/object/0x3c41ff6b5845e0094e19888cba63773591be9de59cafa9e582386f6af15dd490/modules/code/mint_stage?network=mainnet) so it would use the Mainnet token-minter address.
5. Create or get the account you want to create a collection with, open the `.env` file and assign the account address as the `VITE_CREATOR_ADDRESS` value.
6. Finally, run `npm run move:publish` to publish your move module on Aptos mainnet.
7. The next step would be to create a collection using this account. Simply follow [the instructions here](#create-a-nft-collection-page)

## Deploy to a live server

`create-aptos-dapp` utilizes Vite as the development tool. To deploy a Vite static site to a live server, you can simply follow [Vite deployment guide](https://vitejs.dev/guide/static-deploy). In a nutshell, you would need to:
```